lib/MongoDB/Role/_DatabaseOp.pm

```diff
@@ -28,9 +28,12 @@ use Moo::Role;
 
 use MongoDB::_Types qw(
     BSONCodec
+    ClientSession
 );
 use Types::Standard qw(
     Str
+    Maybe
+    InstanceOf
```

Comment:
Do we need InstanceOf?

Reply:
Nope! Missed removing that when we made a top level ClientSession type